api/Dockerfile

```diff
@@ -102,6 +102,7 @@ COPY --chown=opener . /api/
 RUN env \
   SETUP_ES="False" \
   DJANGO_SECRET_KEY="any string" \
+  CANONICAL_DOMAIN="" \
```

Comment:
It needs to be set to _something_, and doesn't need to be empty, but I don't know what makes sense to put here other than an empty string, to convey the fact that it doesn't have any effect on the output of the static files. And, in fact, it _cannot_, otherwise the static files would somehow depend on the environment, and we don't build environment-specific Docker images, so it needs to be environment agnostic.